tests/Tests/500__jupyterhub/culler.robot

```diff
@@ -0,0 +1,169 @@
+*** Settings ***
+Documentation    Tests for the NB culler
+Resource         ../../Resources/ODS.robot
+Resource         ../../Resources/Common.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
+Resource         ../../Resources/Page/ODH/ODHDashboard/ODHDashboard.robot
+Library          ../../../libs/Helpers.py
+Library          OpenShiftLibrary
+Suite Teardown   Teardown
+Force Tags       JupyterHub
+
+
+*** Variables ***
+${DEFAULT_CULLER_TIMEOUT} =    31536000
+${CUSTOM_CULLER_TIMEOUT} =     600
+
+
+*** Test Cases ***
+Verify Default Culler Timeout
+    [Documentation]    Checks default culler timeout
+    [Tags]    Sanity    Tier1
+    ...       ODS-1255
+    Disable Culler
+    ${current_timeout} =  Get Culler Timeout
+    Should Be Equal  ${DEFAULT_CULLER_TIMEOUT}  ${current_timeout}
+    Close Browser
+
+Verify Culler Timeout Can Be Updated
+    [Documentation]    Verifies culler timeout can be updated
+    [Tags]    Sanity    Tier1
+    ...       ODS-1231
+    Modify Culler Timeout    ${CUSTOM_CULLER_TIMEOUT}
+    ${current_timeout} =  Get Culler Timeout
+    Should Not Be Equal  ${current_timeout}  ${DEFAULT_CULLER_TIMEOUT}
+    Should Be Equal   ${current_timeout}  ${CUSTOM_CULLER_TIMEOUT}
+    Close Browser
```

Comment:
should this TC have a teardown to restore the default culler (or disable it) and close browser?

Reply:
It does that in the Suite Teardown

Reply:
related to this other comment https://github.com/red-hat-data-services/ods-ci/pull/295/files/2e86723bff531d72835c322d254fb157dab751e5#r856147372

Reply:
Gotcha, replied over there